src/bun.js/modules/NodeUtilTypesModule2.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#include "BunClientData.h"
+
+JSC_DEFINE_HOST_FUNCTION(jsFunctionIsError,
```

Comment:
Did you mean to commit this?

Reply:
This is for $newCppFn() in util.ts - it can't be done on the existing NodeUtilTypesModule.h because '.h' extension is not allowed, and adding a NodeUtilTypesModule.cpp causes NodeUtilTypesModule.h to get compiled in twice. So this needs to go somewhere, I can put it in a different file though